pycbc/waveform/ringdown.py

```diff
@@ -769,10 +809,186 @@ def get_fd_lm_allmodes(template=None, **kwargs):
+    delta_t = input_params.pop('delta_t', None)
+    t_final = input_params.pop('t_final', None)
+
+    if delta_t is None:
+        delta_t = lm_deltat(f_0, tau, lmns)
+    if t_final is None:
+        t_final = lm_tfinal(tau, lmns)
+
+    kmax = int(t_final / delta_t) + 1
+    # Different overtones will have different tapering window-size
+    # Find maximum window size to create long enough output vector
+    if taper is not None:
+        taper_window = int(taper*max(tau.values())/delta_t)
+        kmax += taper_window
+
+    outplus = TimeSeries(zeros(kmax, dtype=float64), delta_t=delta_t)
+    outcross = TimeSeries(zeros(kmax, dtype=float64), delta_t=delta_t)
+    for lmn in lmns:
+        l, m, nmodes = int(lmn[0]), int(lmn[1]), int(lmn[2])
+        hplus, hcross = get_td_lm(freqs=f_0, taus=tau, l=l, m=m, nmodes=nmodes,
+                             taper=taper, delta_t=delta_t, t_final=t_final,
+                             **input_params)
+        if taper is None:
+            outplus.data += hplus.data
+            outcross.data += hcross.data
+        else:
+            outplus = taper_shift(hplus, outplus)
+            outcross = taper_shift(hcross, outcross)
+
+    return outplus, outcross
+
+def get_fd_freqtau_allmodes(template=None, **kwargs):
+    """Return frequency domain ringdown with all the modes specified.
+    Parameters
+    ----------
+    template: object
+        An object that has attached properties. This can be used to substitute
+        for keyword arguments. A common example would be a row in an xml table.
+    f_lmn: float
+        Central frequency of the lmn overtone, as many as number of modes.
+    tau_lmn: float
+        Damping time of the lmn overtone, as many as number of modes.
+    lmns : list
+        Desired lmn modes as strings (lm modes available: 22, 21, 33, 44, 55).
+        The n specifies the number of overtones desired for the corresponding
+        lm pair (maximum n=8).
+        Example: lmns = ['223','331'] are the modes 220, 221, 222, and 330
+    amp220 : float
+        Amplitude of the fundamental 220 mode.
+    amplmn : float
+        Fraction of the amplitude of the lmn overtone relative to the 
+        fundamental mode, as many as the number of subdominant modes.
+    philmn : float
+        Phase of the lmn overtone, as many as the number of modes.
+    delta_f : {None, float}, optional
+        The frequency step used to generate the ringdown.
+        If None, it will be set to the inverse of the time at which the
+        amplitude is 1/1000 of the peak amplitude (the minimum of all modes).
+    f_lower: {None, float}, optional
+        The starting frequency of the output frequency series.
+        If None, it will be set to delta_f.
+    f_final : {None, float}, optional
+        The ending frequency of the output frequency series.
+        If None, it will be set to the frequency at which the amplitude
+        is 1/1000 of the peak amplitude (the maximum of all modes).
```

Comment:
This? http://ligo-cbc.github.io/pycbc/latest/html/install.html#building-and-installing-documentation
Thanks!
And yeah, I see now what the missing lines are, they must have somehow gotten lost when I copied from other functions...